src/apply-configuration.coffee

```diff
@@ -91,8 +91,8 @@ applyConfiguration = (config) ->
   if config.blueprintPath
     configuration.options.path.push config.blueprintPath
 
-  configuration.options.method.map((method) ->
-    method.toUpperCase()
+  configuration.options.method = (
```

Comment:
Shouldn't it be `configuration.options.methods`? I think name should reflect the fact it's array.